SwiftVoiceQuickstart/ViewController.swift

```diff
@@ -624,19 +621,49 @@ class ViewController: UIViewController, PKPushRegistryDelegate, TVONotificationD
             builder.params = [twimlParamTo : self.outgoingValue.text!]
             builder.uuid = uuid
         }
-        self.call = TwilioVoice.connect(with: connectOptions, delegate: self)
+        let call = TwilioVoice.connect(with: connectOptions, delegate: self)
+        self.activeCalls?.append(call)
         self.callKitCompletionCallback = completionHandler
     }
     
     func performAnswerVoiceCall(uuid: UUID, completionHandler: @escaping (Bool) -> Swift.Void) {
-        let acceptOptions: TVOAcceptOptions = TVOAcceptOptions(callInvite: self.callInvite!) { (builder) in
-            builder.uuid = self.callInvite?.uuid
+        if let callInvite = self.callInvite(uuid: uuid) {
+            let acceptOptions: TVOAcceptOptions = TVOAcceptOptions(callInvite: callInvite) { (builder) in
+                builder.uuid = callInvite.uuid
+            }
+            let call = callInvite.accept(with: acceptOptions, delegate: self)
+            self.activeCalls?.append(call)
+            self.callKitCompletionCallback = completionHandler
+            
+            if let version = Float(UIDevice.current.systemVersion), version < 13.0 {
+                self.incomingPushHandled()
+            }
+            
+            if let index = self.activeCallInvites?.index(of: callInvite) {
+                self.activeCallInvites?.remove(at: index)
```

Comment:
Move the logic of removing the call invite from the list to the CallKit callback `func provider(_ provider: CXProvider, perform action: CXAnswerCallAction)` ? We want to remove in case of error also.